src/Oci8/Oci8ServiceProvider.php

```diff
@@ -58,14 +58,15 @@ public function register()
             }
 
             // set oracle session variables
-            $defaultSessionVars = [
+            $defaultConnection = config('database.default');
+            $sessionVars = [
                 'NLS_TIME_FORMAT'         => 'HH24:MI:SS',
                 'NLS_DATE_FORMAT'         => 'YYYY-MM-DD HH24:MI:SS',
                 'NLS_TIMESTAMP_FORMAT'    => 'YYYY-MM-DD HH24:MI:SS',
                 'NLS_TIMESTAMP_TZ_FORMAT' => 'YYYY-MM-DD HH24:MI:SS TZH:TZM',
                 'NLS_NUMERIC_CHARACTERS'  => '.,',
+                ...config("database.connections.$defaultConnection.sessionVars", []),
```

Comment:
There are cases where Oracle is not the default connection. There are also cases where multiple Oracle connections were configured and will fail on this setup.

Reply:
You're right!
I replaced it with the local variable `$config`.